<?xml version="1.0"?>

<launch>
  <arg name="port" default="/dev/ttySAC0" />

  <node name="driver" pkg="create2_driver" type="create2_driver_node" output="screen">
    <param name="port" value="$(arg port)" />
    <param name="brcPin" value="24" />
    <param name="useBrcPin" value="true" />
  </node>

  <node name="controller" pkg="create2_controller" type="trajectory_controller" output="screen">
    <param name="frame" value="base_link" />
    <param name="parentFrame" value="odom" />
    <param name="Kx" value="5.0" />
    <param name="Ky" value="10.0" />
    <param name="Ktheta" value="40.0" />
  </node>

</launch>
